helm/examples/azure/sa/README.md

```diff
@@ -0,0 +1,70 @@
+- SUBSCRIPTION: Azure subscription
+- STORAGE_ACCOUNT_NAME: Storage Account name which will be generated
+- SA_RESOURCE_GROUP: a resource group name which must be unique in your subscription
+- SA_LOCATION: the region the storage account will be created in. It is recommended to use the same region as the Kubernetes cluster.
+
+hpcc-azurefile/values.yaml defines tree variables needed for storage account and shared storage creation:
+- secretName:  a Kuberentes secret
+- secretNamespace: the namespace of the Kubernetes secret
+- shareName: shared storage name which will be used to create the storage
+
+### Create
+create-sa.sh will source env-sa, create a storage account and shared storage. It will also save the storage key to $SA_KEY_DIR/
+```console
+./create-sa.sh
+```
+### List
+list-sa.sh will list the shared storage
+```console
+./list-sa.sh
+```
+
+
+### Create a Kubernetes secret
+A kubernetes secret is required to use the storage. The secret name, secretName, is defined in hpcc-azurefile/values.yaml.<br/>
+If this Kuberenetes cluster has never generated the secret before, run following to generate the secret
+```console
+./create-secret.sh
+```
+### Install and use hpcc-azurefile
+The hpcc-azurefile helm chart will create the Persistent Volume and the Persistent Volume Claim for each of the required HPCC storage types based on the shared storage.
+Once installed the generated PVC names should be used when installing the HPCC helm chart.
+The examples/azure/values-retained-azurefile.yaml is an example of the HPCC storage settings that should be applied, after the "sa/hpcc-azurefile" helm chart is installed. "azurestorage-hpcc-azurefile" name is expected when launching "helm install" to match the PVC name in the values-retained-azurefile.yaml.
+
+The storage class used in this example supports mountOptions, which are used to ensure that the mounts are owned by user and group 'hpcc' with suitable file permissions.
+
+NB: The output of installing this chart, will contain a generated example with the correct PVC names.
+
+Example use:
+```console
+  helm install azstorage examples/sa/azure/hpcc-azurefile
+```
+
+### values-retained-azurefile.yaml
+An example values file to be supplied when installing the HPCC chart.
+NB: Either use the output auto-generated when installing the "hpcc-azurefile" helm chart, or ensure the names in your values files for the storage types match the PVC names created.
+
+### Delete the storage
+delete-sa.sh will delete the resoruce group which will clean all resources underneath
+```console
+./delete-sa.sh
+```
+You must make certain the contents of the shared storage are no longer needed since when it is deleted you can't get it back.<br/>
+On the other hand, be aware the storage will persist even when the kubernetes cluster is destroyed and there are fees and charges for storage. Be very cautious before making this decision.
```

Comment:
fees and charges for Azure storage.   (I would suggest to specify fees/charges for the Azure storage)